spec/lang/step.md

```diff
@@ -552,14 +552,14 @@ impl<M: Memory> Machine<M> {
         Terminator::CallIntrinsic { intrinsic, arguments, ret: ret_expr, next_block }: Terminator
     ) -> NdResult {
         // First evaluate return place (left-to-right evaluation).
+        let ret_place = ret_expr.try_map(|ret_expr| self.eval_place(ret_expr))?;
```

Comment:
Oh, good point, I might have screwed this up.

We should really have a test covering this...